src/main/webapp/app/text-assessment/text-assessment.component.ts

```diff
@@ -293,6 +297,8 @@ export class TextAssessmentComponent implements OnInit, OnDestroy, AfterViewInit
     public predefineTextBlocks(): void {
         this.assessmentsService.getResultWithPredefinedTextblocks(this.result.id).subscribe(
             response => {
+                const submission = <TextSubmission>response.body!.submission;
```

Comment:
I wonder if your changes have increased the time the server needs for processing the request :thinking: 
It seems like a more complex process to me that might take some time.

I thinkt it would be good to have a loading indicator in the component to show the user the server is still doing work and to prevent multiple successive button presses.